src/core/router/components/Routes.tsx

```diff
@@ -0,0 +1,36 @@
+import { Suspense } from 'react';
+import { CacheRoute } from 'react-router-cache-route';
+import { Route, Redirect } from 'react-router-dom';
+import { useAtom } from '@reatom/react';
+import { availableRoutesAtom } from '../atoms/availableRoutes';
+import { getAbsoluteRoute } from '../routes';
+import { UniversalRoute } from './UniversalRoute';
+import type { AppRouterConfig } from '../types';
+
+const RouterStateToReactRouter = ({ routes }: { routes: AppRouterConfig['routes'] }) => {
+  return (
+    <>
+      {routes.map((r) => (
+        <UniversalRoute
+          key={r.slug}
+          exact
+          path={getAbsoluteRoute(r.parentRoute ? `${r.parentRoute}/${r.slug}` : r.slug)}
+          as={r.cached ? CacheRoute : Route}
+        >
+          <Suspense fallback={<h1>Fallback</h1>}>{r.view}</Suspense>
+        </UniversalRoute>
+      ))}
+    </>
+  );
+};
+
+export function Routes() {
```

Comment:
can we call this component <SelectedRoute />?

Reply:
Why we should name it like that? it's contain all routes

Reply:
Yes...
It's just where i approximately thought routes config transformed to single returned component/view of the route before. (obviously it's not now)

It's something i'd expect to find in a components or just near them. Where is that happening?

Reply:
It logic inside react-router library